Rmds/r_and_rstudio.Rmd

```diff
@@ -0,0 +1,78 @@
+# R and RStudio {-}
+
+To complete the installs of R and RStudio, you must have administrator privileges on your machine. Choose the version appropriate for your operating system. You can also watch a video for a [Mac](https://youtu.be/Oy9BPxt8gQ0?t=270) or a [Windows](https://youtu.be/7f9ADV2v7CI?t=336) install (you only need to install R and RStudio, and none of the other programs mentioned in the videos). Please also note that the versions of R and RStudio in the video are older. Instead, install or update to the latest version supported by your operating system.
+
+1. R https://cran.r-project.org/
+    - version `r getRversion()` (only for macOS Yosemite or older: legacy version `r params$r_legacy_version`)
```

Comment:
Should the sub-bullets be numbered too to show that the order matters?

Reply:
Ok, I will create subheaders and then use numbering instead of bullet points.